src/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -2397,6 +2397,20 @@ elementFormDefault="qualified">
             </xs:complexContent>
         </xs:complexType>
     </xs:element>
+    <xs:element name="GetFileHash" substitutionGroup="msb:Task">
+        <xs:complexType>
+            <xs:complexContent>
+                <xs:extension base="msb:TaskType">
+                    <xs:attribute name="Files" use="required" />
+                    <xs:attribute name="Algorithm" default="SHA256" />
```

Comment:
Is `default` meaningful here? I don't see it in use for any of the other tasks.

Reply:
I was reading about xsd and found this attribute, and in code also we have mentioned the default values, let me know if i should rollback.

Reply:
I'd love to have the extra information, but does it work in Visual Studio? Can you modify your local copy of the XSD and grab a screenshot, maybe?

Reply:
You are right not working , removing the default values.